### PR TITLE
updating ganon to version 0.1.3

### PR DESCRIPTION
**Pull request by pirovc**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/ganon/meta.yaml [hunk 1]
```diff
@@ -18,6 +18,7 @@ source:
     
 build:
   number: 0
+  skip: True # [osx]
```
Comment (pirovc) [inline recipes/ganon/meta.yaml]: osx was skipped due to the lack of gcc support and version of Clang (requires at least 7 and it's current on the version 4)

File: recipes/ganon/meta.yaml [hunk 2]
```diff
@@ -42,10 +49,16 @@ test:
 
 about:
   home: https://github.com/pirovc/ganon
+  summary: "ganon: a read classification tool using Interleaved Bloom Filters"
+  description: |
+          ganon is a k-mer based read classification tool which uses Interleaved Bloom Filters
+          in conjunction with a taxonomic clustering and a k-mer counting-filtering scheme
   license: MIT License
-  summary: 'ganon is a k-mer based read classification tool which uses Interleaved Bloom Filters in conjunction with a taxonomic clustering and a k-mer counting-filtering scheme'
   license_family: MIT
```
Comment (dpryan79) [inline recipes/ganon/meta.yaml]: Can you add `license_file: LICENSE` so the license get's distributed with the package?

Comment (pirovc) [reply recipes/ganon/meta.yaml]: I did so but circleCI cannot find the file even though it's there and I'm not really sure why

Comment (dpryan79) [reply recipes/ganon/meta.yaml]: Very odd. I'll click the "approve" button and we can revert that change

**Pull request merged**